db/migrate/20160820205817_create_publishers.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePublishers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :publications, id: :uuid  do |t|
+      t.string :name
+      t.string :type
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
It was originally publishers, but then I rolledbacked and renamed it
publication.  Couldn't rename the migration and didn't want to create a new
one.  Let me know if there's a better way to this.

On Mon, Aug 22, 2016 at 10:28 PM, Brian Kung <notifications@github.com>
wrote:

> In db/migrate/20160820205817_create_publishers.rb
> <https://github.com/briankung/projectcredo/pull/64#discussion_r75797832>:
>
> > @@ -0,0 +1,10 @@
> > +class CreatePublishers < ActiveRecord::Migration[5.0]
> > +  def change
> > +    create_table :publications, id: :uuid  do |t|
> > +      t.string :name
> > +      t.string :type
> > +
> > +      t.timestamps
> > +    end
> > +  end
> > +end
>
> Are they publishers or publications?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/briankung/projectcredo/pull/64/files/2b8cccb48a7d372bd7c56a40f6732f85f108ff4b#r75797832>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACKsZG9OrPGRh6Jj1Kdy4iip-ooAEuVhks5qimjqgaJpZM4JqR56>
> .
>